packages/sandcastle/src/App.tsx

```diff
@@ -539,6 +540,11 @@ function App() {
       colorScheme={settings.theme}
       synchronizeColorScheme
     >
+      <div className="banner">
+        <Anchor href="https://sandcastle.cesium.com" tone="accent">
+          Switch back to the previous version of Sandcastle
```

Comment:
A thought on wording:

```suggestion
          Looking the old Sandcastle? It’s still here (for a little while) →
```